CHANGELOG.md

```diff
@@ -63,3 +63,26 @@ PYTHON_CONFIGURE_FLAGS = '`"/p:PlatformToolset=clangcl`" `"/p:LLVMInstallDir=C:\
 Runners have a new configuration `use_cores` to control the number of CPU cores
 used to build CPython. By default, this will use all available cores, but some
 Cloud VMs require using fewer.
+
+### Configurable weekly runs
+
+The set of weekly runs are now configured from the `bench_runner.toml`.
+If you don't add any configuration, the only weekly runs performed will be default builds on every runner.
+
+The `weekly` section in the configuration is made up of sections, each of which
+has a `flags` parameter specifying the flags to use and a `runner` parameter
+specifying the runners to run on.
+
+For example:
+
+```toml
+[[weekly]]
```

Comment:
I know I'm late with this review, but I'm confused why `weekly` is an array of tables here. (I'm similarly confused about `[[runners]]` to be fair, but I assumed it was a holdover from an earlier config layout.) Wouldn't it be simpler to leave this out so you don't have the intermediate list, which always contains one item? Or is this just for readability of the config file?

Reply:
I think this is just showing my fuzzy understanding of TOML (and the docs being kind of weird on this point).  I agree it simplifies things to not have the single outer array.  Will fix in another PR.